tests/Doctrine/Tests/DBAL/Driver/StatementIteratorTest.php

```diff
@@ -45,6 +46,8 @@ public function testIteratorIterationCallsFetchOncePerStep() : void
     }
 
     /**
+     * @param class-string<IteratorAggregate &Statement> $class
```

Comment:
The whitespace before `&` looks confusing. `ResultStatement` already extends `Traversable`. Is `IteratorAggregate` really required here?

Reply:
I didn't mean to add the whitespace, sorry. I'll look into it